### PR TITLE
 Poncho is now in the loadout as well, with a little pointer to notify people it can do taur stuff

### PR DESCRIPTION
**Pull request by lbnesquik**
Poncho is also compatible for drake taurs, wolves get janitorial jumpsuit for the future, once uniforms get added to taurify

### CODE REVIEW

**Comment by KasparoVy** [comment]
Cheers

File: code/modules/client/preference_setup/loadout/loadout_suit_vr.dm [hunk 1]
```diff
@@ -32,4 +32,14 @@
...
+	path = /obj/item/clothing/suit/storage/fluff/gntop
+
+/datum/gear/suit/old_poncho //This is made from an old sprite which has been here for quite some time. Called old poncho because duplicates
+	display_name = "Colorful poncho"
+	description = "A mexican looking poncho. It look like it fits wolf taurs as well."
+	path = /obj/item/clothing/suit/poncho
+
+/datum/gear/suit/old_poncho //This is made from an old sprite which has been here for quite some time. Called old poncho because duplicates
+	display_name = "Colorful poncho"
+	description = "A mexican looking poncho. It look like it fits wolf or drake taurs as well."
```
Comment (Heroman3003) [inline code/modules/client/preference_setup/loadout/loadout_suit_vr.dm]: defined twice for some reason, with same path of datum and clothing?

Comment (lbnesquik) [reply code/modules/client/preference_setup/loadout/loadout_suit_vr.dm]: I am not sure what you mean, but there is other ponchos in the game. Particularly for /datum/gear/suit/poncho, so i used old_poncho instead

Comment (Novacat) [reply code/modules/client/preference_setup/loadout/loadout_suit_vr.dm]: There are two listings of /datum/gear/suit/old_poncho, both of them are named exactly the same and have the same contents?

Comment (lbnesquik) [reply code/modules/client/preference_setup/loadout/loadout_suit_vr.dm]: Oh, that's weird.

**Pull request merged**